### ISSUE TITLE
[PABP Figure out how to do roll over image and upload

### ISSUE THREAD

**Issue by daver78** [coa0329/BhaangWeb#141]
Options: 
 https://jsfiddle.net/taditdash/72XyL  // http://jsfiddle.net/k5Gcc/  // https://github.com/Nanakii/summernote-plugins/tree/master/plugin/template // http://talkerscode.com/webtricks/demo/demo_show-button-on-hover-using-html-and-css.php

**Comment by paagla123** [comment]
@daver78 I assume we do not need this one any more - righT?

**Comment by daver78** [comment]
@paagla123 It depends on whether you want to break up the bigger tasks into smaller ones. If you look at the "project" for this overall item, it's one of the to-do items.  Up to you?

**Comment by daver78** [comment]
@paagla123 I'm switching this to you so you can assign to the right person to do what we discussed yesterday:

1- When User rolls over the image, we overlay some text (I can style the overlay and text once it is ready); essentially, it will say something like "Click to edit this image"

2- When the User clicks the Image, the Summernote toolbar appears.

3- In the Summernote toolbar, we need to add a custom button for changing the image.  When you press the button, you can upload a new image or select from the Asset Library.

@paagla123 Confirm this is what we discussed?

**Comment by paagla123** [comment]
@daver78 YES!

**Comment by daver78** [comment]
@paagla123 Who should this be assigned to?

**Comment by paagla123** [comment]
Already assigned to @jiewsone

**Comment by daver78** [comment]
@paagla123 Uh oh you're starting to make use of the infamous "already did BLAH" (and the variant "already done") to mean "I just did it!" :-)

**Comment by paagla123** [comment]
@daver78 it was discussed in one of the calls and @coa0329 assigned to @jiewsone . I just never updated in the issue

**Comment by paweenwich** [comment]
@paagla123 Done,  summernote popup will  have customize upload button (right most button).   Click that button will upload new image to replace the current one.

**Comment by daver78** [comment]
@paweenwich Thank you.  @paagla123 Will you check this today?  Make sure that when we click the button to replace the image, I think there are 3 ways the User may want to operate:

- upload a new image from their desktop
- grab an image from the Asset Library
- paste in a URL to an image already hosted elsewhere

Agree?

**Comment by paagla123** [comment]
@daver78 first one is working. We will fix the rest once ready.

**Comment by daver78** [comment]
@paagla123 What do you mean?  

Do you mean "upload a new image from their desktop" is working, but "grab an image from the Asset Library" and "paste in a URL to an image already hosted elsewhere" isn't working yet?

**Comment by paagla123** [comment]
@daver78 yes. Working for new layout - nice job @jiewsone

**Comment by paagla123** [comment]
Closing this one - I will open a new issue for grabbing image from asset library and adding URL for existing image
